sphinx_js/doclets.py

```diff
@@ -140,6 +143,39 @@ def analyzer_for(language):
         raise SphinxError('Unsupported value of js_language in config: %s' % language)
 
 
+def process_see(doclets):
+    """Process @see tags for all doclets."""
+    for doclet in doclets:
+        if 'see' in doclet:
+            see = doclet['see']
+            process_see_list(see)
```

Comment:
Generally I like to be more explicit that the routine is modifying the passed-in data structure. You could do it through naming the routine differently, but an even better way is to do an assignment here, e.g. `doclet['see'] = process_see_list(...)`.